cmd/main_postsubmit.go

```diff
@@ -0,0 +1,99 @@
+		if err != nil {
+			log.Fatalf("There was an error running make: %v. Make output:\n%v", err, string(output))
+		}
+		fmt.Printf("Output of the make command for %v:\n %v", path, string(output))
+		// Remove the target name from the build args
+		args[1] = root + "/projects/"
+		if path != "coredns/coredns" {
+			command = exec.Command("bash", root+"/release/lib/create_final_dir.sh", releaseBranch, release, artifactBucket, path)
+			output, err = command.CombinedOutput()
+			if err != nil {
+				log.Fatalf("There was an error running the create_final_dir script: %v. Output:\n%v", err, string(output))
+			}
+			fmt.Printf("Output of the create_final_dir script for %v:\n %v", path, string(output))
+			if path == "kubernetes/kubernetes" {
+				command = exec.Command("/bin/bash", "-c", "mv " + root + "/projects/" + path + "/_output/" + releaseBranch + "/*" + " /logs/artifacts")
+			} else {
+				command = exec.Command("/bin/bash", "-c", "mv " + root + "/projects/" + path + "/_output/tar/*" + " /logs/artifacts")
+			}
+			output, err = command.CombinedOutput()
+			if err != nil {
+				log.Fatalf("There was an error running mv: %v. Output:\n%v", err, string(output))
+			}
+			fmt.Printf("Successfully moved artifacts to /logs/artifacts directory for %v.\n", path)
+		}
+	}
+}
+
+func main() {
+	gitRootOutput, err := exec.Command("git", "rev-parse", "--show-toplevel").Output()
+	if err != nil {
+		log.Fatalf("There was an error running the git command: %v", err)
+	}
+	gitRoot := strings.Fields(string(gitRootOutput))[0]
+	kubernetesChanged := false
+	coreDnsChanged := false
+	cniPluginsChanged := false
+	iamAuthChanged := false
+	etcdChanged := false
+	gitDiffCommand := []string{"git", "-C", gitRoot, "diff", "--name-only", "HEAD^", "HEAD"}
+	fmt.Println("\n", strings.Join(gitDiffCommand, " "))
+
+	gitDiffOutput, err := exec.Command("git", gitDiffCommand[1:]...).Output()
+	filesChanged := strings.Fields(string(gitDiffOutput))
+	for _, file := range filesChanged {
+		if strings.Contains(file, "kubernetes/kubernetes") {
+			kubernetesChanged = true
+		}
+		if strings.Contains(file, "coredns/coredns") {
+			coreDnsChanged = true
+		}
+		if strings.Contains(file, "containernetworking/plugins") {
+			cniPluginsChanged = true
+		}
+		if strings.Contains(file, "kubernetes-sigs/aws-iam-authenticator") {
+			iamAuthChanged = true
+		}
+		if strings.Contains(file, "etcd-io/etcd") {
+			etcdChanged = true
+		}
+		if file == "Makefile" {
+			kubernetesChanged = true
+			coreDnsChanged = true
+			cniPluginsChanged = true
+			iamAuthChanged = true
+			etcdChanged = true
+		}
+	}
+	buildArg := []string{"-C", gitRoot + "/projects/", os.Args[1],
+		"RELEASE_BRANCH=" + os.Args[2], "RELEASE=" + os.Args[3],
+		"DEVELOPMENT=" + os.Args[4], "AWS_REGION=" + os.Args[5],
+		"AWS_ACCOUNT_ID=" + os.Args[6], "BASE_IMAGE=" + os.Args[7],
+		"IMAGE_REPO=" + os.Args[8], "IMAGE_TAG='$(GIT_TAG)-$(PULL_BASE_SHA)'"}
+	kubeBuildArg := []string{"-C", gitRoot + "/projects/", os.Args[1],
```

Comment:
It seems like we could consolidate these two since passing extra agurments like the kube-proxy parameters won't harm other make files.